pinc/image_check.inc

```diff
@@ -12,3 +13,14 @@ function get_image_size_error($image_size)
     }
     return $result;
 }
+
+function get_image_small_dimension_error($width, $height)
+{
+    global $page_image_minimum_dimension;
+    $result = null;
+
+    if ($width < $page_image_minimum_dimension || $height < $page_image_minimum_dimension) {
+        $result = sprintf(_("The image may be too small: the shortest dimension of the images is less than %d pixels."), $page_image_minimum_dimension);
```

Comment:
I'd recommend compressing a bit to make the message slightly shorter; maybe: `"Image may be too small: shortest dimension is less than %d pixels."`